lms/services/digest.py

```diff
@@ -36,24 +36,24 @@ def __init__(
         self._email_preferences_service = email_preferences_service
         self._sender = sender
 
-    def send_instructor_email_digests(  # pylint:disable=too-many-arguments
+    def send_instructor_email_digest(  # pylint:disable=too-many-arguments
         self,
-        audience,
+        h_userid,
```

Comment:
The service now generates and sends a single email for a single `h_userid`, rather than a batch.